src/components/ui/ImageGallery/ImageGallerySelector.tsx

```diff
@@ -0,0 +1,130 @@
+import type { HTMLAttributes, PropsWithChildren } from 'react'
+import React, { useMemo } from 'react'
+import { useSlider, IconButton } from '@faststore/ui'
+
+import { ForwardArrowIcon, BackwardArrowIcon } from './Icons'
+
+interface Props extends HTMLAttributes<HTMLDivElement> {
+  itemsPerPage: number
+}
+
+const createTransformValues = (totalItems: number) => {
+  if (!totalItems) {
+    return {}
+  }
+
+  const transformMap: Record<number, number> = {}
+  const slideWidth = 100 / totalItems
```

Comment:
What does this 100 magic number represent? the entire width?

Reply:
Yes, it is the entire width of the element on which the ImageGallery is nested. I'll add a comment to make that explicit.

Reply:
Done at https://github.com/vtex-sites/base.store/pull/142/commits/8f303c5e398e46c7c0396bafbdd32aec813322aa